wordcamp.test/provision/vvv-init.sh

```diff
@@ -20,7 +20,7 @@ WPCLI_THEMES=( twentyten twentyeleven twentytwelve twentythirteen )
 
 wme_svn_git_migration $SITE_DIR
 
-if [ ! -L $SITE_DIR ]; then
+if [ ! -d $SITE_DIR ]; then
```

Comment:
Aren't they supposed to be symlinks? See `wme_symlink_public_dir`.

Reply:
hmm really? 🤔

Reply:
Yeah, that's because Git doesn't allow you to check out a subtree of a repository the way that SVN does. So, we have 1 checkout of the whole `meta.git` repo, and then each site symlinks their `public_html` folder to the corresponding folder inside the repo.